src/client/api/events/sub_outputs_payload.c

```diff
@@ -91,3 +91,16 @@ int event_sub_address_outputs(event_client_handle_t client, int *mid, char const
   }
   return event_subscribe(client, mid, topic_buff, qos);
 }
+
+int event_sub_outputs_id(event_client_handle_t client, int *mid, char const output_id[], int qos) {
+  if ((strlen(output_id)) != OUTPUT_ID_LEN) {
+    printf("[%s:%d]: Output Id length is invalid\n", __func__, __LINE__);
+    return 0;
```

Comment:
should treat it as an error, not return 0.

Reply:
Was a mistake. Returing -1.